std_msgs/Header header
sensor_msgs/RegionOfInterest[] bboxes
sensor_msgs/Image img
recycling_stretch/RotatedBBox[] rbboxes
float64[] scores
int64[] pred_category
float64[] similarity